tests/test_ignore.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+from vulture import core
+
+from . import check, skip_if_not_has_async
+
+
+@pytest.fixture
+def check_ignore():
+    def examine(code, expected, ignore_names=[], ignore_decorators=[]):
```

Comment:
We should always use the kwarg names when passing kwargs to a function. Using kwargs implies that we may reorder the kwargs later. I.e., use `check_ignore(code, ignore_names=['funny'], ignore_decorators=['f?o*', 'ba[rz]'])`. I'd just use `def check_ignore(code, ignore_names, ignore_decorators, expected):` to make things easier and explicit.